db/migrate/20250315194328_enable_pg_stat_statements_extension_v2.rb

```diff
@@ -0,0 +1,9 @@
+class EnablePgStatStatementsExtensionV2 < ActiveRecord::Migration[8.0]
+  def up
+    ApplicationRecord.connection.execute('CREATE EXTENSION pg_stat_statements;')
+  end
+
+  def down
+    ApplicationRecord.connection.execute('DROP EXTENSION pg_stat_statements;')
+  end
```

Comment:
I'm not sure where I came up with the idea for this migration when I added `pghero` last time. It seems that the docs don't suggest it, and instead maybe there's a button to click to enable this extension? But adding it in a migration and having it in the schema seems like a good way to go.